parametric/test_128_bit_traceids.py

```diff
@@ -0,0 +1,310 @@
+import pytest
+
+from parametric.utils.headers import make_single_request_and_get_inject_headers
+from parametric.utils.test_agent import get_span
+
+parametrize = pytest.mark.parametrize
+
+
+@pytest.mark.parametrize(
+    "library_env", [{"DD_TRACE_PROPAGATION_STYLE": "Datadog", "DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": False,}],
```

Comment:
I think you will want both the key and value to be strings:
```suggestion
    "library_env", [{"DD_TRACE_PROPAGATION_STYLE": "Datadog", "DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": "false",}],
```

Reply:
Fixed all instances.